server/src/main/java/io/spine/server/BoundedContextBuilder.java

```diff
@@ -302,6 +338,84 @@ boolean hasRepository(Repository<?, ?> repository) {
         return result;
     }
 
+    /**
+     * Verifies if the repository with a passed entity class was previously added into the
+     * registration list of the Bounded Context this builder is going to build.
+     */
+    @VisibleForTesting
+    <I, E extends Entity<I, ?>> boolean hasRepository(Class<E> entityClass) {
+        checkNotNull(entityClass);
+        boolean result =
+                repositories.stream()
+                            .anyMatch(repository -> repository.entityClass()
+                                                              .equals(entityClass));
+        return result;
+    }
+
+    /**
+     * Adds the passed command dispatcher to the dispatcher registration list which will be
+     * processed after the Bounded Context is created.
+     */
+    @CanIgnoreReturnValue
+    public BoundedContextBuilder addCommandDispatcher(CommandDispatcher<?> commandDispatcher) {
```

Comment:
The tricky business here is that repositories are also `CommandDispatcher`s and `EventDispatcher`s. 

Let's make it impossible to use `[add | remove | has ]...Dispatcher(...)` and ` API with the `...Repository`-typed arguments.